pkgs/development/python-modules/ankipandas/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, pandas
+, colorlog
+, pytest
+, pytest-cov
+, randomfiletree
+}:
+
+buildPythonPackage rec {
+  pname = "ankipandas";
+  version = "0.3.15";
+  format = "setuptools";
```

Comment:
Instead of setting `format` you could do `pyproject = true` here and explicitly add `setuptools` as a build-time dependency.

Note: build-time python deps should no longer go into `nativeBuildInputs = [...]`,
but instead into `build-system = [...]`